stylesheet/utils/display-utils.css

```diff
@@ -1,54 +1,56 @@
-.inline-block {
+.dis-inblock {
     display: inline-block;
 }
 
-.flex-row {
+/* flex display */
+
+.flx {
     display: flex;
+}
+
+.flx-row {
     flex-direction: row;
 }
 
-.flex-column {
-    display: flex;
+.flx-column {
     flex-direction: column;
 }
 
-.flex-order-start {
+.flx-order-start {
     order: -1;
 }
 
-.flex-grow-1 {
+.flx-grow-1 {
     flex-grow: 1;
 }
 
-.flex-center {
-    display: flex;
+.flx-center {
     align-items: center;
     justify-content: center;
 }
 
-.flex-vertical-center {
-    display: flex;
+.flx-min-center {
```

Comment:
suggestion:- I think you should use a bit different naming concept for align-items and justify-content which will be more self-explanatory then flx-min and flx-max , you can use something like jsty-center/ algn-center as without mentioning "flx"   before every class as it is obvious that justify-content/ align-items are used with flex only 

for more clarity you can check various CSS framework/library like tailwind, bootstrap, bulma or others for naming classes

Reply:
I'm using maj and min to represent major and minor axis in flexbox. I have used flx so that user understands that this is a flex property and can be implemented only after using .flx class.